configure.ac

```diff
@@ -1477,7 +1477,7 @@ AC_ARG_WITH(pm,
 
 # --- check and normalize the options ----
 
-case "$with_pmi" in
+case "$with_pmilib" in
```

Comment:
I don't think this is necessarily wrong. Just that is is not well documented in the help string how to use the `--with-pmi` option. MPICH 4.0.x supports `--with-pmi=cray` and internally sets the interface and library version.

Does `--with-pmi=cray` work on Crusher with just the second patch?

Reply:
I see. I was confused by the help string of the `--with-pmi` because it looks like the options for it are `pmi1, pmi2, pmix`. And it seems `--with-pmilib` is the way to set for different PMI libraries.

Maybe I can revise the help string a bit.

Reply:
Oh, it was for backward compatibility. I will remove that commit then.